README.md

```diff
@@ -1,4 +1,4 @@
-# [Google Address Terraform Module](https://registry.terraform.io/modules/terraform-google-modules/address/google/)
+# [Google Address Terraform Module](https://registry.terraform.io/modules/terraform-google-modules/terraform-google-address/)
```

Comment:
Hmm - The new link doesn't seem to work?  I think it should be the original format?